samtranslator/swagger/swagger.py

```diff
@@ -554,10 +554,13 @@ def is_valid(data):
         :param dict data: Data to be validated
         :return: True, if data is a Swagger
         """
-        return bool(data) and \
-            isinstance(data, dict) and \
-            bool(data.get("swagger")) and \
-            isinstance(data.get('paths'), dict)
+        if bool(data) and isinstance(data, dict) and isinstance(data.get('paths'), dict):
+            if bool(data.get("swagger")):
+                return True
+            elif bool(data.get("openapi")):
+                return "3.0" in data["openapi"]
```

Comment:
Same callout about OpenAPI versions (3.0.0 should be supported, I think)

Reply:
3.0.0 is supported, I have a test for it..

Reply:
Oh yeah, this will do string matching. `"3.0"` is inside of `"3.0.0"`. 👍